toolkit/docs/building/building.md

```diff
@@ -210,7 +210,7 @@ To create an unattended ISO installer (no interactive UI) use `UNATTENDED_INSTAL
 
 ```bash
 # Build the standard ISO with unattended installer
```

Comment:
Please extend this comment to clarify that this unattended install configuration expects to run on Gen1 HyperV VMs
```suggestion
# Build the standard ISO with unattended installer that installs onto the default Gen1 HyperV VM
```